service/history/historyEngine.go

```diff
@@ -696,12 +696,18 @@ Update_History_Loop:
 					failCause = workflow.DecisionTaskFailedCause_BAD_START_TIMER_ATTRIBUTES
 					break Process_Decision_Loop
 				}
+				if !userTimersLoaded {
```

Comment:
i just realized the builder is on context which is cached, i need request level sate to reload the state, remembering inside using a flag doesn't work unless i removed it out of the context caching. which seems slightly bigger change. postponing it.